docker/Dockerfile

```
@@ -92,7 +92,9 @@ RUN apt-get -qq update \
     fi \
     # arch specific packages
     && if [ "${TARGETARCH}" = "amd64" ]; then \
-    apt-get -qq install --no-install-recommends --no-install-suggests -y \
+    echo 'deb http://deb.debian.org/debian testing main non-free' >> /etc/apt/sources.list.d/deb.list \
+    && apt-get -qq update \
+    && apt-get -qq install --no-install-recommends --no-install-suggests -y \
```

Does this line need to explicitly use the testing repo?  Can you confirm what version of the driver is installed with the `apt list --installed | grep intel-` command?  
```suggestion
    && apt-get -qq -t testing install --no-install-recommends --no-install-suggests -y \
```

---

This is with the way my PR is now:

```
intel-gpu-tools/now 1.26+git20220524-1 amd64 [installed,local]
intel-media-va-driver-non-free/now 22.6.1+ds1-1 amd64 [installed,local]
mesa-va-drivers/now 22.2.0-1 amd64 [installed,local]
```

This is with your suggested change:

```
intel-gpu-tools/now 1.26+git20220524-1 amd64 [installed,local]
intel-media-va-driver-non-free/now 22.6.1+ds1-1 amd64 [installed,local]
mesa-va-drivers/now 22.2.0-1 amd64 [installed,local]
```

---

Ok, interesting.  Does that mean all packages afterward could also be installed from testing?

---

Right, just those few related to hwaccel which I think is ideal, shouldn't affect anything else.

---

Oh, this is the last apt install on amd64.  No worries then, thanks